selfdrive/car/car_specific.py

```diff
@@ -86,6 +86,23 @@ def update(self, CS: car.CarState, CS_prev: car.CarState, CC: car.CarControl):
       if self.CP.minEnableSpeed > 0 and CS.vEgo < 0.001:
         events.add(EventName.manualRestart)
 
+      # Low speed steer alert logic; for cars with steer cut off above 6 m/s
+      # Show the alert earlier when slowing. Must first exceed a greater speed while engaged.
+      if CS.out.vEgo > (self.CP.minSteerSpeed + 3.5):
+        CS.min_steer_alert_speed = (self.CP.minSteerSpeed + 1.5)
+      elif not CC.latActive:
+        CS.min_steer_alert_speed = self.CP.minSteerSpeed
+      self.low_speed_alert = (0 < CS.out.vEgo <= CS.min_steer_alert_speed) and self.CP.minSteerSpeed > 6.0
+      if self.low_speed_alert:
+        events.add(EventName.belowSteerSpeed)
```

Comment:
Can you move all this into the interface and use lowSpeedAlert?

Reply:
FYI @csouers this is the refactor I was talking about above. There's a good example to follow in #33712, converting Mazda to use the new common mechanism.

Reply:
Nice. Done.

As for the steer_off_cnt, i need a way to get the prev latActive or similar inside of carstate. Not seeing a way as things are.

Reply:
oh wait! i see a loopback can parser is a thing. neat. will try that